README.md

```diff
@@ -354,7 +360,7 @@ Set hosts file:
 ```sh
 echo "127.0.0.1 localhost
 ::1 localhost
-127.0.1.1 arch.localdomain arch" >> /etc/hosts
+127.0.0.1 arch.localdomain arch" >> /etc/hosts
```

Comment:
See https://wiki.archlinux.org/index.php/Installation_guide#Network_configuration

Reply:
I stand corrected. Thanks for the pointer!